### DISCUSSION TITLE
Assistance for Deleting Outdated Content

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Knowlg/Community#178] [General Discussions]
Hi Team,

We are currently using Sunbird version 5.0.0 and require assistance in the process to **find out the content which is not consumed** from more than an year and **want to delete that content**. Please provide us with instructions for this procedure.

cc: @raghupathiguduri

**Comment by raghupathiguduri** [comment]
@AmiableAnil Can you please help us here ? 

We want to delete the contents belongs to one channel/organisation.

Also we have below queries regarding 5.1.0 c-name migrations. 

We want to retire contents before migration, Will the migration jobs ignore the retired content during the process ?

**Reply by pallakartheekreddy** [reply]
@raghupathiguduri @Venkatesh-Dayani23 When consuming any content, the START telemetry event is generated by players in sunbird. Utilizing this event allows us to identify content that has not been consumed within a specific time.

**Reply by pallakartheekreddy** [reply]
@raghupathiguduri As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
